client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -521,6 +527,14 @@ export default {
       return this.assignee.id === this.currentUserId
     },
 
+    hasRecommendationTabs () {
+      const recommendationTabs = ['TagRecommendationTab', 'OracleRecommendationTab']
+
+      return this.loadedAddons.some(addon =>
+        recommendationTabs.some(tab => addon.name.includes(tab))
+      )
```

Comment:
this makes not so much sense imho. loadedAddons are always recommendationTab-Addons. To clearify that, I recommend to rename `this.loadedAddons`  to `recommendationModalAddons` (see above). Then this check would look like 
```
    hasRecommendationTabs () {
       return this.recommendationModalAddons.length > 0
   }
```